src/world/common/enemy/FlyingAI.inc.c

```diff
@@ -0,0 +1,32 @@
+#ifndef AI_PRREQ_FLYING
+#define AI_PRREQ_FLYING 0
```

Comment:
If this is just an include guard we typically write it like this
```suggestion
#ifndef _AI_PRREQ_FLYING_H_
#define _AI_PRREQ_FLYING_H_
```

Reply:
Technically it's not good practice to do _X because it's considered reserved. I think it's fine to do it like this, and we should maybe migrate the others later